TCK server AccountInfoQuery fails when run against `hiero-sdk-tck` test

### Description

When running the Hiero SDK TCK against the Java tck  server, the `AccountInfoQuery` test fails.


### Steps to reproduce

1. Start the TCK server for Java
2. Run the test for AccountInfoQuery in `hiero-sdk-tck`
```
npm run test:file src/tests/crypto-service/test-account-info-query.ts
```

### Additional context

```bash
    32) (#37) Query account info and verify stakingInfo.stakedAccountId
      33) (#38) Query account info and verify stakingInfo.stakedNodeId


  5 passing (2m)
  33 failing

  1) AccountInfoQuery
       Account ID
         (#1) Query for the info of a valid account:
     AssertionError: expected {} to have property 'accountId'
      at Context.<anonymous> (src/tests/crypto-service/test-account-info-query.ts:46:32)
      at processTicksAndRejections (node:internal/process/task_queues:105:5)

  2) AccountInfoQuery
       Account ID
         (#5) Query account info and verify accountId is returned:
     AssertionError: expected undefined to equal '0.0.7950931'
      at Context.<anonymous> (src/tests/crypto-service/test-account-info-query.ts:115:37)
      at processTicksAndRejections (node:internal/process/task_queues:105:5)

  3) AccountInfoQuery
       Account ID
         (#6) Query account info and verify contractAccountId is returned:
     AssertionError: expected undefined to exist
      at Context.<anonymous> (src/tests/crypto-service/test-account-info-query.ts:129:44)
      at processTicksAndRejections (node:internal/process/task_queues:105:5)

  4) AccountInfoQuery
       Account ID
         (#7) Query account info and verify isDeleted is false:
     AssertionError: expected undefined to be false
      at Context.<anonymous> (src/tests/crypto-service/test-account-info-query.ts:144:39)
      at processTicksAndRejections (node:internal/process/task_queues:105:5)

  5) AccountInfoQuery
       Account ID
         (#9) Query account info and verify proxyAccountId:
     AssertionError: expected {} to have property 'proxyAccountId'
      at Context.<anonymous> (src/tests/crypto-service/test-account-info-query.ts:187:32)
      at processTicksAndRejections (node:internal/process/task_queues:105:5)

  6) AccountInfoQuery
       Account ID
         (#10) Query account info and verify proxyReceived:
     AssertionError: expected {} to have property 'proxyReceived'
      at Context.<anonymous> (src/tests/crypto-service/test-account-info-query.ts:201:32)
      at processTicksAndRejections (node:internal/process/task_queues:105:5)

```

### Hedera network

testnet

### Version

unrelease

### Operating system

Linux

--- a/tck/src/main/java/com/hedera/hashgraph/tck/methods/sdk/response/ContractResponse.java
+++ b/tck/src/main/java/com/hedera/hashgraph/tck/methods/sdk/response/ContractResponse.java
@@ -2,86 +2,46 @@
 package com.hedera.hashgraph.tck.methods.sdk.response;
 
 import com.hedera.hashgraph.sdk.Status;
-import java.util.LinkedHashMap;
-import java.util.Map;
-import net.minidev.json.JSONAware;
-import net.minidev.json.JSONObject;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
-public record ContractResponse(String contractId, Status status) {
-    public record ContractInfoQueryResponse(
-            String contractId,
-            String accountId,
-            String contractAccountId,
-            String adminKey,
-            String expirationTime,
-            String autoRenewPeriod,
-            String autoRenewAccountId,
-            String storage,
-            String contractMemo,
-            String balance,
-            Boolean isDeleted,
-            String maxAutomaticTokenAssociations,
-            String ledgerId,
-            ContractResponse.ContractInfoQueryResponse.StakingInfoResponse stakingInfo)
-            implements JSONAware {
-        public record StakingInfoResponse(
-                Boolean declineStakingReward,
-                String stakePeriodStart,
-                String pendingReward,
-                String stakedToMe,
-                String stakedAccountId,
-                String stakedNodeId)
-                implements JSONAware {
-            @Override
-            public String toJSONString() {
-                JSONObject json = new JSONObject();
-                if (declineStakingReward != null) {
-                    json.put("declineStakingReward", declineStakingReward);
-                }
-                if (stakePeriodStart != null) {
-                    json.put("stakePeriodStart", stakePeriodStart);
-                }
-                if (pendingReward != null) {
-                    json.put("pendingReward", pendingReward);
-                }
-                if (stakedToMe != null) {
-                    json.put("stakedToMe", stakedToMe);
-                }
-                if (stakedAccountId != null) {
-                    json.put("stakedAccountId", stakedAccountId);
-                }
-                if (stakedNodeId != null) {
-                    json.put("stakedNodeId", stakedNodeId);
-                }
-                return json.toJSONString();
-            }
-        }
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ContractResponse {
+    private String contractId;
+    private Status status;
 
-        @Override
-        public String toJSONString() {
-            JSONObject json = new JSONObject();
-            Map<String, Object> values = new LinkedHashMap<>();
-            values.put("contractId", contractId);
-            values.put("accountId", accountId);
-            values.put("contractAccountId", contractAccountId);
-            values.put("adminKey", adminKey);
-            values.put("expirationTime", expirationTime);
-            values.put("autoRenewPeriod", autoRenewPeriod);
-            values.put("autoRenewAccountId", autoRenewAccountId);
-            values.put("storage", storage);
-            values.put("contractMemo", contractMemo);
-            values.put("balance", balance);
-            values.put("isDeleted", isDeleted);
-            values.put("maxAutomaticTokenAssociations", maxAutomaticTokenAssociations);
-            values.put("ledgerId", ledgerId);
-            values.put("stakingInfo", stakingInfo);
+    @Data
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class ContractInfoQueryResponse {
+        private String contractId;
+        private String accountId;
+        private String contractAccountId;
+        private String adminKey;
+        private String expirationTime;
+        private String autoRenewPeriod;
+        private String autoRenewAccountId;
+        private String storage;
+        private String contractMemo;
+        private String balance;
+        private Boolean isDeleted;
+        private String maxAutomaticTokenAssociations;
+        private String ledgerId;
+        private StakingInfoResponse stakingInfo;
 
-            values.forEach((key, value) -> {
-                if (value != null) {
-                    json.put(key, value);
-                }
-            });
-            return json.toJSONString();
+        @Data
+        @AllArgsConstructor
+        @NoArgsConstructor
+        public static class StakingInfoResponse {
+            private Boolean declineStakingReward;
+            private String stakePeriodStart;
+            private String pendingReward;
+            private String stakedToMe;
+            private String stakedAccountId;
+            private String stakedNodeId;
         }
     }
 }

--- a/tck/src/main/java/com/hedera/hashgraph/tck/methods/sdk/response/GetAccountInfoResponse.java
+++ b/tck/src/main/java/com/hedera/hashgraph/tck/methods/sdk/response/GetAccountInfoResponse.java
@@ -3,81 +3,132 @@ package com.hedera.hashgraph.tck.methods.sdk.response;
 
 import java.util.List;
 import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 /**
  * Response for getAccountInfo query
  */
-public record GetAccountInfoResponse(
-        String accountId,
-        String contractAccountId,
-        boolean isDeleted,
-        String proxyAccountId,
-        String proxyReceived,
-        String key,
-        String balance,
-        String sendRecordThreshold,
-        String receiveRecordThreshold,
-        boolean isReceiverSignatureRequired,
-        String expirationTime,
-        String autoRenewPeriod,
-        List<LiveHashResponse> liveHashes,
-        Map<String, TokenRelationshipInfo> tokenRelationships,
-        String accountMemo,
-        String ownedNfts,
-        String maxAutomaticTokenAssociations,
-        String aliasKey,
-        String ledgerId,
-        List<HbarAllowanceResponse> hbarAllowances,
-        List<TokenAllowanceResponse> tokenAllowances,
-        List<TokenNftAllowanceResponse> nftAllowances,
-        String ethereumNonce,
-        com.hedera.hashgraph.tck.methods.sdk.response.GetAccountInfoResponse.StakingInfoResponse stakingInfo) {
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class GetAccountInfoResponse {
+    private String accountId;
+    private String contractAccountId;
+    private boolean isDeleted;
+    private String proxyAccountId;
+    private String proxyReceived;
+    private String key;
+    private String balance;
+    private String sendRecordThreshold;
+    private String receiveRecordThreshold;
+    private boolean isReceiverSignatureRequired;
+    private String expirationTime;
+
+    // Explicit getters to ensure proper JSON serialization
+    public boolean getIsDeleted() {
+        return isDeleted;
+    }
+
+    public boolean getIsReceiverSignatureRequired() {
+        return isReceiverSignatureRequired;
+    }
+
+    private String autoRenewPeriod;
+    private List<LiveHashResponse> liveHashes;
+    private Map<String, TokenRelationshipInfo> tokenRelationships;
+    private String accountMemo;
+    private String ownedNfts;
+    private String maxAutomaticTokenAssociations;
+    private String aliasKey;
+    private String ledgerId;
+    private List<HbarAllowanceResponse> hbarAllowances;
+    private List<TokenAllowanceResponse> tokenAllowances;
+    private List<TokenNftAllowanceResponse> nftAllowances;
+    private String ethereumNonce;
+    private StakingInfoResponse stakingInfo;
+
     /**
      * LiveHashResponse for account info
      */
-    public record LiveHashResponse(String accountId, String hash, List<String> keys, String duration) {}
+    @Data
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class LiveHashResponse {
+        private String accountId;
+        private String hash;
+        private List<String> keys;
+        private String duration;
+    }
 
     /**
      * TokenRelationshipInfo for account info
      */
-    public record TokenRelationshipInfo(
-            String tokenId,
-            String symbol,
-            String balance,
-            Boolean isKycGranted,
-            Boolean isFrozen,
-            Boolean automaticAssociation) {}
+    @Data
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class TokenRelationshipInfo {
+        private String tokenId;
+        private String symbol;
+        private String balance;
+        private Boolean isKycGranted;
+        private Boolean isFrozen;
+        private Boolean automaticAssociation;
+    }
 
     /**
      * HbarAllowanceResponse for account info
      */
-    public record HbarAllowanceResponse(String ownerAccountId, String spenderAccountId, String amount) {}
+    @Data
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class HbarAllowanceResponse {
+        private String ownerAccountId;
+        private String spenderAccountId;
+        private String amount;
+    }
 
     /**
      * TokenAllowanceResponse for account info
      */
-    public record TokenAllowanceResponse(
-            String tokenId, String ownerAccountId, String spenderAccountId, String amount) {}
+    @Data
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class TokenAllowanceResponse {
+        private String tokenId;
+        private String ownerAccountId;
+        private String spenderAccountId;
+        private String amount;
+    }
 
     /**
      * TokenNftAllowanceResponse for account info
      */
-    public record TokenNftAllowanceResponse(
-            String tokenId,
-            String ownerAccountId,
-            String spenderAccountId,
-            List<String> serialNumbers,
-            Boolean allSerials,
-            String delegatingSpender) {}
+    @Data
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class TokenNftAllowanceResponse {
+        private String tokenId;
+        private String ownerAccountId;
+        private String spenderAccountId;
+        private List<String> serialNumbers;
+        private Boolean allSerials;
+        private String delegatingSpender;
+    }
 
     /**
      * StakingInfoResponse for account info
      */
-    public record StakingInfoResponse(
-            boolean declineStakingReward,
-            String stakePeriodStart,
-            String pendingReward,
-            String stakedToMe,
-            String stakedAccountId,
-            String stakedNodeId) {}
+    @Data
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class StakingInfoResponse {
+        private boolean declineStakingReward;
+        private String stakePeriodStart;
+        private String pendingReward;
+        private String stakedToMe;
+        private String stakedAccountId;
+        private String stakedNodeId;
+    }
 }

